<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiChannelHomesAd</class>
 <widget class="QDialog" name="uiChannelHomesAd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>135</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>405</width>
    <height>135</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>135</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Changing channels</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/logo_64.png</normaloff>:/images/logo_64.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_msg">
      <property name="text">
       <string>Message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_channel"/>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
